client/workitemtype.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
+	"net/http"
+	"net/url"
+)
+
+// ShowWorkitemtypePath computes a request path to the show action of workitemtype.
+func ShowWorkitemtypePath(id string) string {
```

Comment:
The reason why it shows up as lowercase here is most likely because the Resource in the design is named all lowercase: https://github.com/almighty/almighty-core/pull/4/files#diff-5147d84da6b10c041a0bafd5935f9f4fR39